<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Tamanna" ignored="2" name="Default test(failed)(failed)" tests="1" failures="1" timestamp="2021-03-20T13:19:09 BDT" time="0.27" errors="0">
  <testcase name="test3" time="0.048" classname="assertion_TestNG.Retry_rerun">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at assertion_TestNG.Retry_rerun.test3(Retry_rerun.java:17)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test2" time="0.0" classname="assertion_TestNG.Retry_rerun">
    <ignored/>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.0" classname="assertion_TestNG.Retry_rerun">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed)(failed) -->
